{{ cookiecutter.library_name }}/.github/workflows/build.yml

```diff
@@ -59,3 +59,9 @@ jobs:
     - name: Build docs
       working-directory: docs
       run: sphinx-build -E -W -b html . _build/html
+    - name: Build README
```

Comment:
I'd call this `Build python package` since it'll fail if the setup.py steps fail. Otherwise, this looks good to me.

Reply:
Ok, I've renamed it. Thanks for taking a look at this